magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/create_LST_table.py

```diff
@@ -0,0 +1,61 @@
+"""
+Create a new h5 table from the one of joint observations.
+
+Only the columns needed to produce the lists of LST runs to be processed are presenved, and two columns are added to store NSB level and error codes
+"""
+
+import os
+
+import numpy as np
+import pandas as pd
+
+
+def main():
+
+    """
+    Main function
+    """
+
+    df = pd.read_hdf(
+        "/fefs/aswg/workspace/federico.dipierro/simultaneous_obs_summary.h5", key="/str"
+    )  # TODO: put this file in a shared folder
+
+    needed_cols = [
+        "source",
+        "DATE",
+        "LST1_run",
+        "MAGIC_stereo",
+        "MAGIC_trigger",
+        "MAGIC_HV",
+    ]
+    df_cut = df[needed_cols]
+
+    df_cut["nsb"] = np.repeat(np.nan, len(df_cut))
+
+    df_cut["lstchain_0.9"] = np.zeros(len(df_cut), dtype=bool)
+
+    df_cut["lstchain_0.10"] = np.zeros(len(df_cut), dtype=bool)
+
+    df_cut["error_code"] = np.repeat(np.nan, len(df_cut))
```

Comment:
as we discussed the error codes we can specify for individual programs

Reply:
fixed